sdk/appconfiguration/azure-mgmt-appconfiguration/CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Release History
 
+## 1.0.0b1 (2020-06-17)
+
+This is beta preview version.
+For detailed changelog please refer to equivalent stable version 0.4.0 (https://pypi.org/project/azure-mgmt-appconfiguration/0.4.0/)
+
+This version uses a next-generation code generator that introduces important breaking changes, but also important new features (like unified authentication and async programming).
+
+## General breaking changes
```

Comment:
this is the same level with version declaration, would it be better to one level deeper:
###.....

Reply:
yup, you're right :-)